[JIT] Fixes to prim ops

eellison opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#32179 [JIT] Fixes to prim ops**

Tensors are used as keys in dictionaries, so we need to annotate that key insertion into a dictionary inserts the key into the wildcard set. Also fixes bug with `listCopyAndSort` not copying the input list.

Differential Revision: [D19397555](https://our.internmc.facebook.com/intern/diff/D19397555)

Commit:
fix schemas

[ghstack-poisoned]

kostmo commented:
## :pill: CircleCI build failures summary and remediations
As of commit ea2d1094:


* **1/1** failures introduced in this PR


## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=ea2d1094c7ac6bde2f3ef366814bf0304f05d23c).

### :detective: 1 new failure recognized by patterns
The following build failures do not appear to be due to upstream breakage:
#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4240956) pytorch_xla_linux_xenial_py3_6_clang7_build (1/1)
**Step:** "Build" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=85996064) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=85996064))
<details>
<summary>
<code>Jan 14 19:45:33 RuntimeError: Failed to apply patch: /var/lib/jenkins/workspace/xla/torch_patches/X10-clip_grad.diff</code>
</summary>

```
Jan 14 19:45:33 + XDIR=/var/lib/jenkins/workspace/xla/scripts/.. 
Jan 14 19:45:33 + PTDIR=/var/lib/jenkins/workspace/xla/scripts/../.. 
Jan 14 19:45:33 + python /var/lib/jenkins/workspace/xla/scripts/cond_patch.py /var/lib/jenkins/workspace/xla/scripts/../torch_patches /var/lib/jenkins/workspace/xla/scripts/../.. 
Jan 14 19:45:33 1 out of 1 hunk FAILED 
Jan 14 19:45:33 Applying patch file: /var/lib/jenkins/workspace/xla/torch_patches/X10-clip_grad.diff 
Jan 14 19:45:33 Traceback (most recent call last): 
Jan 14 19:45:33   File "/var/lib/jenkins/workspace/xla/scripts/cond_patch.py", line 67, in <module> 
Jan 14 19:45:33     patch_repo(args) 
Jan 14 19:45:33   File "/var/lib/jenkins/workspace/xla/scripts/cond_patch.py", line 49, in patch_repo 
Jan 14 19:45:33     raise RuntimeError('Failed to apply patch: {}'.format(ppath)) 
Jan 14 19:45:33 RuntimeError: Failed to apply patch: /var/lib/jenkins/workspace/xla/torch_patches/X10-clip_grad.diff 
Jan 14 19:45:33 + cleanup 
Jan 14 19:45:33 + retcode=1 
Jan 14 19:45:33 + set +x 
Jan 14 19:45:33 =================== sccache compilation log =================== 
Jan 14 19:45:33 =========== If your build fails, please take a look at the log above for possible reasons =========== 
Jan 14 19:45:33 Compile requests                 1 
Jan 14 19:45:33 Compile requests executed        0 
Jan 14 19:45:33 Cache hits                       0 
Jan 14 19:45:33 Cache misses                     0 
Jan 14 19:45:33 Cache timeouts                   0 
```

</details>



---

<details><summary>This comment was automatically generated by <a href="https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment">Dr. CI</a> (expand for details).</summary>Follow <a href="https://dr.pytorch.org/admin/comments-opt-out.html">this link to opt-out</a> of these comments for your Pull Requests.

Please report bugs/suggestions on the <a href="https://github.com/kostmo/circleci-failure-tracker/issues">GitHub issue tracker</a>.
</details>

Commit:
Update on "[JIT] Fixes to prim ops"


Tensors are used as keys in dictionaries, so we need to annotate that key insertion into a dictionary inserts the key into the wildcard set. Also fixes bug with `listCopyAndSort` not copying the input list. 


[ghstack-poisoned]

Commit:
Update on "[JIT] Fixes to prim ops"


Tensors are used as keys in dictionaries, so we need to annotate that key insertion into a dictionary inserts the key into the wildcard set. Also fixes bug with `listCopyAndSort` not copying the input list. 


[ghstack-poisoned]
